hedyorg/hedy
# 🪲 Fixes accessing deleted tags on adventure

Checks if the tag is different than `None`.

Fixes #5579  

**How to test**

The best way would be to open an adventure, add a tag, and then delete it in the DB. Reload the server and you should be able to keep editing the adventure.
